scripts/tests/perf/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -xe
+
+mkdir -p /tmp/iperf3results
+docker compose exec --env RUST_LOG=info -it client /bin/sh -c 'iperf3 -t 30 -O 1 -R -c 172.20.0.110 --json' >>/tmp/iperf3results/tcp_server2client.json
+cat /tmp/iperf3results/tcp_server2client.json | jq -r '"tcp_server2client_sum_received_bits_per_second=" + (.end.sum_received.bits_per_second|tostring)' >>"$GITHUB_OUTPUT"
+cat /tmp/iperf3results/tcp_server2client.json | jq -r '"tcp_server2client_sum_sent_bits_per_second=" + (.end.sum_sent.bits_per_second|tostring)' >>"$GITHUB_OUTPUT"
+cat /tmp/iperf3results/tcp_server2client.json | jq -r '"tcp_server2client_sum_sent_retransmits=" + (.end.sum_sent.retransmits|tostring)' >>"$GITHUB_OUTPUT"
```

Comment:
If you:

- Move all the `cat` to the bottom of this file
- Parameterise `compare.sh` with the path to the directory of the JSON files

Then you can de-duplicate a lot of code here by calling `compare.sh` from this script.

Reply:
Turns out we can just remove this entirely :-)